### DISCUSSION TITLE
ImportError: cannot import name 'SQLAlchemyBaseOAuthAccountTable' from 'fastapi_users.db' (/usr/local/lib/python3.9/site-packages/fastapi_users/db/__init__.py)

### DISCUSSION THREAD

**Discussion by EssaAlshammri** [fastapi-users/fastapi-users#811] [Q&A] [answered]
for some reason this just got thrown although it's been working before.

if I change it from 

``` python
from fastapi_users.db import (
    SQLAlchemyBaseOAuthAccountTable,
    SQLAlchemyBaseUserTable,
    SQLAlchemyUserDatabase,
)
```
to 

``` python
from fastapi_users_db_sqlalchemy import (
    SQLAlchemyBaseOAuthAccountTable,
    SQLAlchemyBaseUserTable,
    SQLAlchemyUserDatabase,
)
```

it just works 

the only thing I did is in reqirements.txt

it was like this:

```
fastapi-users[sqlalchemy, oauth]
```

and became like:

```
fastapi-users[sqlalchemy, oauth]==8.1.2
```

even vs code doesn't autocomplete :)

![Screenshot 2021-11-28 144344](https://user-images.githubusercontent.com/10750698/143766369-d01568da-25e4-40f5-b8f3-f7f1b2ab92d1.png)

Am I missing something?

**Comment by EssaAlshammri** [comment]
on the interpreter it just works 

but when run with uvicorn it throws

**Comment by frankie567** [comment, marked as answer]
I've verified on a clean virtual environment, and it works. Be sure to run the interpreter and Uvicorn from a clean virtual environment and try to reinstall your dependencies.
